README.md

```diff
@@ -5,7 +5,23 @@ auto-entities
 
 Automatically populate lovelace cards with entities matching certain criteria.
 
-For installation instructions [see this guide](https://github.com/thomasloven/hass-config/wiki/Lovelace-Plugins).
+## Installation
+
+### HACS
+
+Search for for `auto-entities` in Plugins. After installing restart Home Assistant.
+
+In order to use it in the UI, add the following in the `resources` section
+
+```yaml
+resources:
+- url: /community_plugin/lovelace-auto-entities/auto-entities.js
```

Comment:
`community_plugin` is wrong, that should be `hacsfiles`

Reply:
At least for the docker HA installation, `community_plugin` is the right path
```
resources:
- type: js
  url: /community_plugin/rgb-light-card/card.js
- url: /community_plugin/lovelace-auto-entities/auto-entities.js
  type: js
```